cmd/nerdctl/run_cgroup_linux.go

```diff
@@ -148,6 +148,23 @@ func generateCgroupOpts(cmd *cobra.Command, id string) ([]oci.SpecOpts, error) {
 		}
 		opts = append(opts, oci.WithCPUCFS(cpuQuota, cpuPeriod))
 	}
+	cpuRTRuntime, err := cmd.Flags().GetInt64("cpu-rt-runtime")
+	if err != nil {
+		return nil, err
+	}
+	cpuRTPeriod, err := cmd.Flags().GetUint64("cpu-rt-period")
+	if err != nil {
+		return nil, err
+	}
+	if cpuRTRuntime != -1 || cpuRTPeriod != 0 {
+		if cpus > 0.0 {
+			return nil, errors.New("cpus and RT runtime/period should be used separately")
+		}
+		if cpuQuota != -1 || cpuPeriod != 0 {
+			return nil, errors.New("quota/period and RT runtime/period should be used separately")
```

Comment:
I'm not 100% sure if this is true. Also not sure if need to add a check of `cpuRTRuntime < cpuRTRuntime`